### PR TITLE
Falcosidekick servicemonitor port and selector fix

### PR DESCRIPTION
**Pull request by fa-at-pulsit**
**What type of PR is this?**

/kind bug

/kind chart-release

**Any specific area of the project related to this PR?**

/area falcosidekick-chart

**What this PR does / why we need it**:

fix: mismatch port name in servicemonitor
fix: added component core to selector, overwise prometheus tries to select also falcosidekick-ui (if installed) and fails 

**Which issue(s) this PR fixes**:

Fixes #

fix: mismatch port name in servicemonitor
fix: added component core to selector, overwise prometheus tries to select also falcosidekick-ui (if installed) and fails 

**Special notes for your reviewer**:

**Checklist**

- [X] Chart Version bumped
- [X] CHANGELOG.md updated

### CODE REVIEW

**Review by Issif — APPROVED** [review]
Nice catch. Thanks

**Comment by Issif** [comment]
Can you sign off your commit please

**Comment by fa-at-pulsit** [comment]
@Issif  somehow my sign off is not recognised... should I rebase it and commit again or can you force it?

**Comment by Issif** [comment]
Some of your commits are not signed off. You should squash them, sign them off and push force. A rebase could be useful too.

File: charts/falcosidekick/CHANGELOG.md [hunk 1]
```diff
@@ -5,604 +5,608 @@ numbering uses [semantic versioning](http://semver.org).
 
 Before release 0.1.20, the helm chart can be found in `falcosidekick` [repository](https://github.com/falcosecurity/falcosidekick/tree/master/deploy/helm/falcosidekick).
 
+## 0.7.13
+
+- Fix ServiceMonitor port name and selector labels
+
 ## 0.7.12
 
-* Align README values with the values.yaml file
```
[the remark this answers is not in the document]
Comment (fa-at-pulsit) [reply charts/falcosidekick/CHANGELOG.md:10]: Yes, sure, sorry about that.

**Comment by tberreis** [comment]
I may be wrong but isn't the label app.kubernetes.io/component now set twice?
For example it's added to the deployment [here](https://github.com/falcosecurity/charts/blob/master/charts/falcosidekick/templates/deployment.yaml#L9) and also to the `falcosidekick.labels` [here](https://github.com/falcosecurity/charts/blob/master/charts/falcosidekick/templates/_helpers.tpl#L66).

**Pull request merged**